src/ui/src/Validation/validateCreateAuction/validateCreateAuction.ts

```diff
@@ -0,0 +1,35 @@
+import dayjs from 'dayjs';
+import auctionTypes from '../../domain/enums/SmartAuctionType';
+import { ICreateAuctionFormValues } from '../../interfaces/Forms/CreateAuction/ICreateAuctionFormValues';
+
+function validateCreateAuction(values: ICreateAuctionFormValues): any {
+  const errors: any = {};
+
+  if (!Object.values(auctionTypes).includes(values?.auctionType)) {
+    errors.auctionType = `Unavailable auction type: available types are: ${Object.keys(
+      auctionTypes
+    ).join(', ')}\n`;
+  }
+
+  if (values?.description?.length >= 200) {
+    errors.description += 'Description maximum length is 200 characters\n';
+  }
+
+  if (values?.name?.length < 3 || values?.name?.length > 100) {
+    errors.name += 'Name length should be between 3 and 100 characters\n';
+  }
+
+  if (!values?.name) {
+    errors.name += 'Name is required\n';
+  }
+
+  if (!values?.startTime) {
+    errors.startTime += 'Start time is required\n';
+  } else if (dayjs().diff(values.startTime) > 0) {
```

Comment:
think it's better to put dayjs into the current variable const currentTime = dayjs();
   if (currentTime.diff(startTime) > 0) {
so it will be clearer that in the condition

Reply:
That'd be a good idea if we didn't have the explaining validation message located 1 line below